components/mail/dovecot/dovecot.p5m

```diff
@@ -0,0 +1,1055 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Josef 'Jeff' Sipek <jeffpc@josefsipek.net>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+<transform file path=usr/libexec/dovecot/.* -> default mode 0555>
```

Comment:
On Thu, Sep 08, 2016 at 20:45:48 -0700, Alexander Pyhalov wrote:
> > +#
> > +
> > +#
> > +# Copyright 2016 Josef 'Jeff' Sipek <jeffpc@josefsipek.net>
> > +#
> > +
> > +set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
> > +set name=pkg.summary value="$(COMPONENT_SUMMARY)"
> > +set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
> > +set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
> > +set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
> > +set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
> > +
> > +license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
> > +
> > +<transform file path=usr/libexec/dovecot/.* -> default mode 0555>
> 
> We don't have /usr/libexec, use /usr/lib

Hrm, this will be interesting - dovecot uses both /usr/lib/dovecot and
/usr/libexec/dovecot.  The first is for shared libs, and the second for
helper executables it forks & execs.  I don't really want to mix the two, so
I'll move the executables to /usr/lib/dovecot/foo/.

-- 
The obvious mathematical breakthrough would be development of an easy way to
factor large prime numbers.
		- Bill Gates, The Road Ahead, pg. 265